docs/DeveloperGuide.md

```diff
@@ -660,52 +705,43 @@ testers are expected to do more *exploratory* testing.
 1. Initial launch
 
    1. Download the jar file and copy into an empty folder
-
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
-
-1. Saving window preferences
+   2. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
```

Comment:
from line 708 onwards, i see that `newline` has been deleted between every numbered step e.g (1. something \n 2. something), personally think it is good to have the `newline` for clean visual.

note: My observation is having the `newline` makes a difference in webpage but not sure about IDE, the difference between having `newline` or not may not be shown in IDE

Reply:
My apologies, I failed to realize that my IDE automatically removed the blank lines and changed the numbers. 😨
Will make amendments! 🥺